rustlang/Cargo.toml

```diff
@@ -4,13 +4,6 @@ version = "0.1.0"
 authors = ["Stan Rozenraukh <stan@stanistan.com>"]
 edition = "2018"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
-num-format = "0.4.0"
-warp = "0.2"
-url = "2.1.1"
-reqwest = { version = "0.10", features = ["json"] }
-tokio = { version = "0.2", features = ["full"] }
+airtable-proxy = { path = "airtable-proxy" }
 serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
```

Comment:
1. 

We explicitly clean up the dependencies, moving them into the proxy (or whatever happened here).